ui/components/multichain/asset-picker-amount/asset-picker-amount.stories.tsx

```diff
@@ -45,6 +45,34 @@ TokenStory.decorators = [
   ),
 ];
 
+export const NFTStory = () => (
```

Comment:
I'd love for us to add more stories:

1.  Mixture of NFTs and tokens
2. ~10 tokens
3. Token with really long name and values.

Reply:
done :) , regarding the point 1 and 2 , this will be add in the multiselect tokens/nfts modal